### PR TITLE
Remove duplication of the labextension

### PR DESCRIPTION
**Pull request by martinRenou**
Taking bqplot as a reference implementation for packaging the labextension. Thanks @maartenbreddels who implemented this approach.

This removes the duplication of the labextension in the wheel (which basically divises by two the size of the wheel for xeus-python-kernel, going from ~30MB to ~15MB).

@jtpio if you think this approach makes sense, I'm happy to upstream those changes into `jupyterlab/extension-cookiecutter-ts` and fix https://github.com/jupyterlab/extension-cookiecutter-ts/issues/220

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,10 +33,11 @@
     "build:dockerimage_no_cache": "docker build -t mydockerimage --no-cache . ",
     "build:emscripten": "echo $(pwd) && docker run --rm -v $(pwd):/src -u $(id -u):$(id -g) mydockerimage copy_output.sh",
     "copy-files": "copyfiles -u 1 src/xpython_wasm.wasm src/xpython_wasm.worker.js src/xpython_wasm.js src/python_data.js src/python_data.data lib",
+    "copy-packagejson": "copyfiles package.json jupyterlite_xeus_python/",
```
Comment (martinRenou) [inline package.json:36]: This is the only thing that I find annoying in this PR. Having to copy the `package.json` to be installed under `site-packages` feels dirty.

Comment (martinRenou) [reply package.json:36]: Technically nothing stops us from having the version separated between the Python and the NPM package. That is what we do with jupyter-widgets. If we keep those versions separated, we don't need the `package.json` around the Python code anymore.

**Comment by martinRenou** [comment]
Actually we can get rid of the `package.json` under site-packages if we follow what bqplot does all the way through (finding the `package.json` in the prefix).

File: package.json (hunk 1 above)
Comment (martinRenou) [reply package.json]: Undone this `package.json`. It's not needed.

**Comment by maartenbreddels** [comment]
> Thanks @maartenbreddels who implemented this approach.

Thanks :) What PR is this referring to btw?

**Comment by martinRenou** [comment]
It was for the nbextension in bqplot https://github.com/bqplot/bqplot/pull/1228 and we followed together the same pattern for the labextension https://github.com/bqplot/bqplot/pull/1222 :)

**Pull request merged**